src/main/java/uk/gov/companieshouse/web/lfp/service/latefilingpenalty/impl/LateFilingPenaltyServiceImpl.java

```diff
@@ -28,6 +31,9 @@ public class LateFilingPenaltyServiceImpl implements LateFilingPenaltyService {
 
     private static final String PENALTY_TYPE = "penalty";
 
+    protected static final Logger LOGGER = LoggerFactory
```

Comment:
I reckon, since there is no subclass, this LOGGER can be declared private scope.